scale/scheduler/scheduling/manager.py

```diff
@@ -367,10 +368,22 @@ def _process_queue(self, nodes, job_types, job_type_limits, job_type_resources,
             for resource in job_exe.required_resources.resources:
                 # skip sharedmem
                 if resource.name.lower() == 'sharedmem':
```

Comment:
`sharedmem` isn't a resource that is recognized by Mesos and therefore outputting on warning on it is meaningless. `sharedmem` is only a configuration that is offered by way of our `resources` job-type config that is passed into Docker as a special parameter.

Reply:
That said the if block needs to remain so we skip marking `sharedmem` as a required resource and don't try to tell Mesos our job needs the non-existent `sharedmem` resource.